### PR TITLE
Add support for visualization_msgs Marker and MarkerArray

### PR DESCRIPTION
**Pull request by KavitShah1998**

### CODE REVIEW

File: ros_tcp/ros_communication_client/ros_communications.py [hunk 1]
```diff
@@ -326,9 +335,34 @@ def show_image(nparr):
...
         "/new_image_pub", "sensor_msgs/Image", callback_fn=show_image
     )
-
     while time.time() - start_time <= 60:
         time.sleep(1)
     subscriber.unsubscribe()
     stop_publishing = True
     publisher_thread.join()
+
+    # image = np.zeros((480, 640, 3), dtype=np.uint8)
```
Comment (jimmytyyang) [inline ros_tcp/ros_communication_client/ros_communications.py:344]: remove this?

File: ros_tcp/ros_communication_client/ros_communications.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# mypy: ignore-errors
```
Comment (jimmytyyang) [inline ros_tcp/ros_communication_client/ros_communications.py:1]: remove this?

File: ros_tcp/ros_communication_client/ros_tcp.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# mypy: ignore-errors
```
Comment (jimmytyyang) [inline ros_tcp/ros_communication_client/ros_tcp.py:1]: remove this?

**Review by jimmytyyang — COMMENTED** [review]
left some small comments. Thanks for working on this!

File: ros_tcp/ros_communication_client/ros_tcp.py (hunk 3 above)
Comment (KavitShah1998) [reply ros_tcp/ros_communication_client/ros_tcp.py]: removed

File: ros_tcp/ros_communication_client/ros_communications.py (hunk 2 above)
Comment (KavitShah1998) [reply ros_tcp/ros_communication_client/ros_communications.py]: removed

File: ros_tcp/ros_communication_client/ros_communications.py (hunk 1 above)
Comment (KavitShah1998) [reply ros_tcp/ros_communication_client/ros_communications.py]: removed

**Review by jimmytyyang — APPROVED** [review]
LGTM!

**Review by jimmytyyang — APPROVED** [review]
LGTM!

**Pull request merged**